gpxpy/parser.py

```diff
@@ -165,7 +165,7 @@ def __init__(self, xml_or_file=None, parser=None):
 
     def init(self, xml_or_file):
         text = xml_or_file.read() if hasattr(xml_or_file, 'read') else xml_or_file
-        if text[:3] == "\xEF\xBB\xBF": #Remove utf-8 Byte Order Mark (BOM) if present
+        if text.startswith(u'\xEF\xBB\xBF'):  # Remove utf-8 Byte Order Mark (BOM) if present
```

Comment:
I'm even more confused now. make_str actually makes a unicode when run under Python 3 and a byte when run under Python 2. Why?

@tkrajina please help